.npmrc

```diff
@@ -0,0 +1,3 @@
+registry=https://pkgs.dev.azure.com/devdiv/_packaging/Pylance/npm/registry/ 
```

Comment:
You need this in order to pull down pylance from the internal feed. After this is set you run the directions supplied here to connect:
https://dev.azure.com/devdiv/DevDiv/_packaging?_a=connect&feed=Pylance